package.json

```diff
@@ -8,7 +8,7 @@
     "browser-test-remote-firefox": "nightwatch --config nightwatch.js --env default",
     "browser-test-remote-ie": "nightwatch --config nightwatch.js --env ie",
     "browser-test-remote-safari": "nightwatch --config nightwatch.js --env safari",
-    "build": "mkdir -p build; browserify src/index.js -g yo-yoify -o build/app.js; babel --plugins babel-plugin-transform-es2015-template-literals,babel-plugin-transform-es2015-block-scoping build/app.js --out-file build/app.js",
+    "build": "mkdir -p build; browserify src/index.js -g yo-yoify -o build/app.js; babel --plugins babel-plugin-transform-es2015-template-literals,babel-plugin-transform-es2015-block-scoping build/app.js --out-file build/app.js; lessc --clean-css assets/less/styles.less build/app.css",
```

Comment:
Don't use `less --clean-css` because that compacts the file and `csslint` will not be very useful.